Sources/AppCoinsSDK/UI/Purchase/ViewModels/TransactionViewModel.swift

```diff
@@ -130,8 +131,8 @@ internal class TransactionViewModel : ObservableObject {
             }
         }
     }
-
-    private func getTransactionBonus(address: String, domain: String, amount: String, currency: Coin, completion: @escaping (TransactionBonus) -> Void) {
+    
+    private func getTransactionBonus(address: String, domain: String, amount: String, currency: String, completion: @escaping (TransactionBonus) -> Void) {
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done